.github/workflows/e2e-tests.yml

```
@@ -0,0 +1,194 @@
+        working-directory: web
+        run: npx playwright test --project=${{ matrix.browser }}
+
+      - name: Upload browser-specific test results
+        uses: actions/upload-artifact@v4
+        if: always()
+        with:
+          name: e2e-test-results-${{ matrix.browser }}
+          path: |
+            web/playwright-report/
+            web/test-results/
+          if-no-files-found: ignore
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)